core/nwb.ophys.yaml

```diff
@@ -121,46 +119,53 @@ groups:
       dtype: text
       name: description
       quantity: '?'
+    - doc: a table for storing pixel masks
+      name: pixel_mask_weights
+      dtype:
+      - name: x
+        dtype: uint
+        doc: the pixel x-coordinate
+      - name: y
+        dtype: uint
+        doc: the pixel y-coordinate
+      - name: y
```

Comment:
should this be named ``weight`` instead of ``y``?

Reply:
you are correct